src/core_plugins/input_control_vis/public/control/list_control_factory.js

```diff
@@ -61,12 +68,12 @@ class ListControl extends Control {
     const indexPattern = this.filterManager.getIndexPattern();
     const fieldName = this.filterManager.fieldName;
     const initialSearchSourceState = {
-      timeout: '1s',
-      terminate_after: 100000
+      timeout: `${_.get(this.options, 'timeout', DEFAULT_TIMEOUT)}s`,
+      terminate_after: _.get(this.options, 'terminateAfter', DEFAULT_TERMINATE_AFTER)
```

Comment:
Same comment here, I think `terminate_after` is a bit misleading. `max_result_set_size` or `max_num_docs` or something might be clearer. Also, should it be camelCased instead of snake_cased?

Reply:
I have to use `terminate_after` here because it is an Elasticsearch parameter, https://www.elastic.co/guide/en/elasticsearch/reference/current/search-request-body.html#_parameters_4. 

But I can change the label and description in the UI. Although, it might make sense to stay with Elasticsearch naming schemes since there are really just ways to allow users to configure elasticsearh parameters.

Reply:
Yup. Agreed that it's best to stick w/ the Elasticsearch terminology. I wasn't aware of that!